game/src/main/kotlin/world/gregs/voidps/world/interact/entity/player/combat/consume/drink/Overload.kts

```diff
@@ -11,17 +13,32 @@ import world.gregs.voidps.engine.timer.timerStart
 import world.gregs.voidps.engine.timer.timerStop
 import world.gregs.voidps.engine.timer.timerTick
 import world.gregs.voidps.world.interact.entity.combat.hit.directHit
+import world.gregs.voidps.world.interact.entity.combat.inWilderness
 import world.gregs.voidps.world.interact.entity.player.combat.consume.canConsume
 
-fun inWilderness() = false
-
 canConsume("overload*") { player ->
+    if (player.inWilderness) {
+        player.message("You cannot drink an overload potion while you're in the wilderness.", ChatType.Game)
+        cancel()
+        return@canConsume
+    }
+
     if (player.timers.contains("overload")) {
         player.message("You may only use this potion every five minutes.")
         cancel()
+        return@canConsume
     } else if (player.levels.get(Skill.Constitution) < 500) {
         player.message("You need more than 500 life points to survive the power of overload.")
         cancel()
+        return@canConsume
+    }
+}
+
+enterArea("wilderness*") {
```

Comment:
It should just be `wilderness` the others are multi areas which can extend outside of the actual pvp area, see; map viewer
![image](https://github.com/GregHib/void/assets/5911414/fa5cf2a0-6cca-42ae-b86f-c4e6204bed6a)

Reply:
I am unable to get the map viewer to work properly for me. what i see is a black screen with the areas outlined i will attached a screenshot. @GregHib (sorry for the ping but just wanted to make sure you see this)
![map](https://github.com/GregHib/void/assets/79747812/367afb11-4fc8-434d-9059-29e541d40106)

Reply:
under world.gregs.voidps.tools.map.render you need to run WorldMapDumper.kt to dump the images then run map viewer

Reply:
> under world.gregs.voidps.tools.map.render you need to run WorldMapDumper.kt to dump the images then run map viewer

Thank you, didn't know i had to run that before running the viewer